### PR TITLE
Fixed typos and messages

### PR DESCRIPTION
**Pull request by anilsingla**
Fixed typos and messages in attestation

### CODE REVIEW

File: src/main/bash/commands/besman-attest.sh [hunk 1]
```diff
@@ -24,13 +24,13 @@ function __bes_attest {
 	export COSIGN_PASSWORD=$(openssl rand -base64 32)
         export COSIGN_KEY_LOCATION=$(pwd)
 
-        cosign version 2>&1>/dev/null
+        which cosign 2>&1>/dev/null
```
Comment (asa1997) [inline src/main/bash/commands/besman-attest.sh:27]: Use this instead,
```code
if [[ -z $(which cosing) ]];
then
   if code
else
   else code
fi
```
or

```code
if [[ -z $(command -v cosing) ]];
then
   if code
else
   else code
fi
```

File: src/main/bash/commands/besman-verify.sh [hunk 2]
```diff
@@ -23,13 +23,14 @@ function __bes_verify {
         fi
 
 	#check if cosign installed
-	cosign version 2>&1>/dev/null
+	which cosign 2>&1>/dev/null
```
Comment (asa1997) [inline src/main/bash/commands/besman-verify.sh:26]: Use this instead,
```code
if [[ -z $(which cosing) ]];
then
   if code
else
   else code
fi
```
or

```code
if [[ -z $(command -v cosing) ]];
then
   if code
else
   else code
fi
```

File: src/main/bash/commands/besman-verify.sh [hunk 3]
```diff
@@ -40,22 +41,22 @@ function __bes_verify {
         fi
 
         #verify the signature
-	__besman_echo_yellow "Verifying signature for $filename..."
-        cosign verify-blob $filename --key cosign.pub --bundle $filename.bundle 2>&1 | tee signresult > /dev/null
-
-
+        cosign verify-blob $filename --key cosign.pub --bundle $filename.bundle 2>&1 | tee sigresult > /dev/null
```
Comment (asa1997) [inline src/main/bash/commands/besman-verify.sh:44]: Remove the hardcoding of sigresult. Use a variable with absolute path

File: src/main/bash/commands/besman-verify.sh [hunk 4]
```diff
@@ -40,22 +41,22 @@ function __bes_verify {
...
 	#verify the attestation
 	cosign verify-blob-attestation $filename --key cosign.pub --bundle $filename.attest.bundle 2>&1 | tee attestresult > /dev/null
 
-        sigr=$(cat signresult)
+        sigr=$(cat sigresult)
 	attr=$(cat attestresult)
 
-	if grep "Verified OK" signresult 2>&1>/dev/null &&  grep "Verified OK" signresult 2>&1>/dev/null ;then
+	if grep "Verified OK" sigresult 2>&1>/dev/null &&  grep "Verified OK" attestresult 2>&1>/dev/null ;then
```
Comment (asa1997) [inline src/main/bash/commands/besman-verify.sh:51]: Remove the hardcoding of attestresult. Use a variable with absolute path

File: src/main/bash/commands/besman-attest.sh [hunk 5]
```diff
@@ -24,13 +24,13 @@ function __bes_attest {
...
 
 	if [ xx"$?" != xx"0" ];then
 	  # install COSIGN
           LATEST_VERSION=$(curl https://api.github.com/repos/sigstore/cosign/releases/latest | grep tag_name | cut -d : -f2 | tr -d "v\", ")
-          curl -O -L "https://github.com/sigstore/cosign/releases/latest/download/cosign_${LATEST_VERSION}_amd64.deb"
-          sudo dpkg -i cosign_${LATEST_VERSION}_amd64.deb
+          curl --silent -O -L "https://github.com/sigstore/cosign/releases/latest/download/cosign_${LATEST_VERSION}_amd64.deb" 2>&1>/dev/null
```
Comment (asa1997) [inline src/main/bash/commands/besman-attest.sh:32]: See if you can go for a besman wrapper function under utils.

**Review by asa1997 — CHANGES_REQUESTED** [review]

File: src/main/bash/commands/besman-attest.sh (hunk 5 above)
Comment (anilsingla) [reply src/main/bash/commands/besman-attest.sh:30]: we need -O flag which is not present in wrapper functions so can not use the wrapper functions

**Review by asa1997 — APPROVED** [review]
@anilsingla LGTM

**Pull request merged**